src/client/__init__.py

```diff
@@ -2564,14 +2568,14 @@ def get_task_suites(self, request: search_requests.TaskSuiteSearchRequest) -> Ge
     @expand('patch')
     @add_headers('client')
     def patch_task_suite(self, task_suite_id: str, patch: task_suite.TaskSuitePatch) -> TaskSuite:
-        """Changes the task suite overlap or priority
+        """Changes task suite parameters in Toloka.
```

Comment:
Are we changing parameters here or their values? Maybe it's better to write something like `Changes task suite parameter values in Toloka.`?